### ISSUE TITLE
Gio DOES have a conllu treebank file for Gregory Nazianzenus

### ISSUE THREAD

**Issue by gregorycrane** [scaife-viewer/beyond-translation-site#190]
https://drive.google.com/file/d/1sDgFuWQlwyGchrvgBznxMRgD1rY6F9lk/view?usp=drive_link

**Comment by jacobwegner** [comment]
@gregorycrane Yes, I was aware.  I'm continuing to "round-robin" through these datasets.

You can view the trees now:

https://preview.scaife-viewer.eldarion.com/nyssa-di-russo-trees
